README.md

```diff
@@ -23,7 +23,98 @@ yarn add @paraswap/sdk
 
 ## Using ParaSwap SDK
 
-### Import the necessary functions
+There are multiple ways to use ParaSwap SDK, ranging from simple construct-and-use approach to a fully composable _bring what you need_ way that allows for advanced tree-shaking and minimizes bundle size.
+
+### Simple SDK
+
+Can be created by providing `network` and either `axios` or `window.fetch` (or alternative `fetch` implementation). Resulting SDK will be able to use all methods that query the API.
+
+```ts
+  import { constructSimpleSDK } from '@paraswap/sdk'
+  import axios from 'axios'
+
+  // construct minimal SDK with fetcher only
+  const paraSwapMin = constructSimpleSDK({network: 1, axios})
+  // or
+  const paraSwapMin = constructSimpleSDK({network: 1, fetch: window.fetch})
+
+  async function swapExample() {
+    const signer: JsonRpcSigner = ...
+    const senderAddress = signer.address
+
+    const priceRoute = await paraSwapMin.getRate({
+      srcToken: ETH,
+      destToken: DAI,
```

Comment:
perhaps [define](https://github.com/paraswap/paraswap-sdk/blob/sdk-refactoring_simple_helpers/tests/simple.test.ts#L28-L29) these above or stringify (eg `'ETH'` `'DAI'`) to leave no uncertainty for the user/dev.

Reply:
Done [ab6bf97](https://github.com/paraswap/paraswap-sdk/pull/123/commits/ab6bf976a5130b611ea55eaca64bb08f46fbef55)